src/test/test.spec.js

```diff
@@ -0,0 +1,219 @@
+import chai from 'chai';
+import Cryptr from 'cryptr';
+import dotenv from 'dotenv';
+import chaiHttp from 'chai-http';
+import app from '../index';
+import generateToken from '../helpers/generateToken';
+import helpers from '../helpers/helpers';
+
+dotenv.config();
+chai.use(chaiHttp);
+
+const { should, expect } = chai;
+should();
+const cryptr = new Cryptr(process.env.JWT_SECRET);
+const faketoken = cryptr.encrypt('iamfaketokendonttrustme');
+
+describe('TEST ALL ENDPOINT', () => {
+  describe('Initial testing', () => {
+    it('should return welcome to sims', done => {
+      chai
+        .request(app)
+        .get('/')
+        .end((err, res) => {
+          res.body.should.be.an('object');
+          res.body.message.should.eql('Welcome to the Kifaru backend page');
+          done();
+        });
+    });
+  });
+
+  describe('TESTING HELPERS FUNCTION', () => {
+    it('should validate password', () => {
+      const value = helpers.validPassword('PassWord');
+      expect(value.valid).to.equal(true);
+    });
+    it('should validate email false', () => {
+      const value = helpers.validEmail('email');
+      expect(value).to.equal(false);
+    });
+    it('should validate email', () => {
+      const value = helpers.validEmail('email@email.com');
+      expect(value).to.equal(true);
+    });
+  });
+});
+
+describe('Password reset test', () => {
+  it('should return success message', done => {
+    chai
+      .request(app)
+      .post('/api/v1/auth/reset-password')
+      .send({
+        email: 'timitejumola@andela.com'
+      })
+      .end((err, res) => {
+        res.should.have.status(200);
+        res.body.should.have.property('message');
+        res.body.should.have.property('status');
+        res.body.status.should.be.equal('success');
+        res.body.message.should.be.equal(
+          'Password reset link has been sent to your email'
```

Comment:
Change the message to `A password reset link would be sent to the email provided if it is associated with a registered account` or something similar.